packages/docs/src/pages/components.js

```diff
@@ -33,13 +33,13 @@ const ComponentGroup = ({ groupName, components }) => {
       key={slugify(component.node.frontmatter.title)}
       title={component.node.frontmatter.title}
       html={component.node.html}
-      id={`${groupName.replace(' ', '-')}-${component.node.frontmatter.title.replace(' ', '-')}`}
+      id={`${slugify(groupName, '-')}-${slugify(component.node.frontmatter.title, '-')}`}
```

Comment:
nit: You can do
```
slugify(`${groupName} ${component.node.frontmatter.title}`, '-');
```
unless you wanted to save/cache the value of `slugify(groupName, '-')` outside of this map to not compute it every time and to use it on line 42.

Reply:
Similarly in `ComponentSideMenu`